subgraph.yaml

```diff
@@ -5,12 +5,12 @@ features:
     - ipfsOnEthereumContracts
 dataSources:
     - kind: ethereum
-      name: FontProjectV2
+      name: FontProject
       network: mumbai
       source:
           address: '0x94f24dbf0c81006284c76ba59bbabc5a24de1817'
-          abi: FontProjectV2
-          startBlock: 29547385
+          abi: FontProject
+          startBlock: 30585687
```

Comment:
Is this the block taken from the last proxy?

Reply:
This block number takes only the last two test transactions for the sake of speeding up indexing...
<img width="1437" alt="image" src="https://user-images.githubusercontent.com/761087/212674322-c1514568-009b-42d4-94f9-bc91a5e5a11d.png">